rct229/rulesets/ashrae9012019/section11/section11rule16.py

```diff
@@ -0,0 +1,90 @@
+from rct229.rule_engine.rule_base import RuleDefinitionBase
+from rct229.rule_engine.rule_list_indexed_base import RuleDefinitionListIndexedBase
+from rct229.rule_engine.ruleset_model_factory import produce_ruleset_model_description
+from rct229.rulesets.ashrae9012019 import BASELINE_0
+from rct229.schema.schema_enums import SchemaEnums
+
+ENERGY_SOURCE = SchemaEnums.schema_enums["EnergySourceOptions"]
+
+MANUAL_CHECK_REQUIRED_MSG = (
+    "The baseline service water heating has propane as a fuel source. "
+    "Natural gas is the required fuel source for the baseline model except in cases where natural gas is not available on-site. "
+    "Verify that natural gas is not available for the proposed building site as determined by the rating authority."
+)
+
+
+class Section11Rule16(RuleDefinitionListIndexedBase):
+    """Rule 16 of ASHRAE 90.1-2019 Appendix G Section 11 (Service Water Heating)"""
+
+    def __init__(self):
+        super(Section11Rule16, self).__init__(
+            rmds_used=produce_ruleset_model_description(
+                USER=False, BASELINE_0=True, PROPOSED=True
+            ),
+            each_rule=Section11Rule16.SWHEquipRule(),
+            index_rmd=BASELINE_0,
+            id="11-16",
+            description="Gas water heaters shall be modeled using natural gas as their fuel. Exceptions: Where natural gas is not available for the proposed building site, as determined by the rating authority, gas water heaters shall be modeled using propane as their fuel.",
+            ruleset_section_title="Service Water Heating",
+            standard_section="Table G3.1 #11, baseline column, (h)",
+            is_primary_rule=True,
+            list_path="ruleset_model_descriptions[0].service_water_heating_equipment[*]",
+        )
+
```

Comment:
Here is missing a `create_data` to get the `proposed_fuels`

```python
def create_data(self, context, data):
    rmd_p = context.PROPOSED
    return {
        "proposed_fuels": find_all("$.ruleset_model_descriptions[0].service_water_heating_equipment[*].heater_fuel_type)
   }
```
You do not need PROPOSED in the `SWHEquipRule` level but simply take out this list.

Reply:
Addressed.